### ISSUE TITLE
Move EC2 instance to a VPC

### ISSUE THREAD

**Issue by waldoj** [openva/richmondsunlight#63]
Towards #62:

* [ ] create VPC-based RDS instance
* [ ] shut down Apache on the live EC2 instance
* [ ] dump live RDS and load it into VPC-based RDS instance
* [ ] create an AMI based on the live EC2 instance
* [ ] make sure [EC2 and RDS instances are in the same security group](https://linuxacademy.com/blog/linux/converting-your-aws-ec2-rds-instances-to-amazon-vpc/)
* [ ] launch that AMI into the VPC
* [ ] reassign the live Elastic IP to the VPC-based EC2 instance
* [ ] delete old EC2 instance

**Comment by waldoj** [comment]
I think there's an easier way: enabling ClassicLink. That will allow this to be done in stages. But the catch is that [linking the EC2 instance to the VPC](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/vpc-classiclink.html#classiclink-link-instance) isn't working. The option is grayed out in the EC2 config. The only apparently prerequisites are that the EC2 be running under Classic (it is), that the account also support VPC (it does), and that the two be running in the same region (they are). I tried using the AWS CLI, but setting that up became rage-inducing, and I had to stop.
